elm.json

```
@@ -1,11 +1,9 @@
 {
-    "version": "1.0.0",
+    "type": "package",
+    "name": "circuithub/elm-semantic-ui",
     "summary": "helpful summary of your project, less than 80 characters",
```

Brilliant. Let me help out. 75 chars.
```suggestion
     "summary": "A reasonable type safe interface to creating Semantic UI based applications",
```

---

thanks!